### PR TITLE
console debug log level

### PR DESCRIPTION
**Pull request by KaelynJefferson**
**Description:** This PR adds a debug log level option to the configuration settings menu to allow a user to toggle the debug level messages that appear in the console.

**Testing Instructions:** Run jest unit tests and view within browser.

**Related Issue:** n/a

### CODE REVIEW

File: tests/components/FSHControls.test.jsx [hunk 1]
```diff
@@ -231,16 +236,95 @@ it('calls GoFSH with the indent option if the configuration checkbox is checked'
...
+      },
+      'info'
+    ); // No IG resource added because canonical and version set to defaults
     expect(onGoFSHClick).toHaveBeenCalledTimes(2);
     expect(onGoFSHClick).toHaveBeenCalledWith('', true); // Loading
     expect(onGoFSHClick).toHaveBeenCalledWith(simpleFsh, false);
   });
 });
 
+it('calls GoFSH with the logger level debug if the configuration checkbox is checked', async () => {
```
Comment (mint-thompson) [inline tests/components/FSHControls.test.jsx:253]: Should there also be a test for running SUSHI with debug logging enabled?

**Review by mint-thompson — COMMENTED** [review]
This looks good! I tried it out locally, and sure enough, I see the expected logging messages when I change the option. I have one small question about the tests.

**Review by cmoesel — APPROVED** [review]
This is a nice feature! I've reviewed the code and tested it manually. Debug logging works for both SUSHI and GoFSH. Thanks, @KaelynJefferson!

**Review by mint-thompson — APPROVED** [review]
Thanks for adding that test!

**Pull request merged**